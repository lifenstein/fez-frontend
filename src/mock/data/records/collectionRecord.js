import { hydrateMock } from '../../../helpers/general';

const collectionRecord = {
    rek_pid: 'UQ:11398',
    rek_title: 'This is a Collection Record',
    rek_display_type: 9,
    rek_display_type_lookup: 'Collection',
    rek_status: 2,
    rek_status_lookup: 'Published',
    rek_object_type: 2,
    rek_object_type_lookup: 'Collection',
    "rek_citation": "<a class=\"citation_collections\" title=\"Click to list records in Collection Aboriginal+and+Torres+Strait+Islander+Studies+Unit+Publications\" href=\"/records/search?searchQueryParams%5Brek_ismemberof%5D%5Bvalue%5D%5B%5D=UQ%3A11398&searchMode=advanced\">Aboriginal+and+Torres+Strait+Islander+Studies+Unit+Publications</a>",
    "fez_record_search_key_ismemberof": [
        {
            "rek_ismemberof": "UQ:12096",
            "parent": {
                "rek_pid": "UQ:12096",
                "rek_security_policy": 1,
                "rek_datastream_policy": null
            },
            "rek_ismemberof_lookup": "Aboriginal and Torres Strait Islander Studies Unit"
        }
    ],
    "fez_record_search_key_keywords": [
        "Torres Strait Islanders",
        "Aborigines",
    ],
    fez_record_search_key_xsd_display_option: [263],
    rek_wok_doc_type_lookup: 'Abstract of Published Item',
    "ciNotices": {
        "rek_ci_notice_attribution_incomplete":true,
    },
};
export default hydrateMock(collectionRecord);
