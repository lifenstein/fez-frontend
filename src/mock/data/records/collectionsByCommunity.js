export default {
    "total": 3,
    "took": 19,
    "per_page": 20,
    "current_page": 1,
    "from": 1,
    "to": 3,
    "data": [{
    "rek_pid": "UQ:11398",
    "rek_title_xsdmf_id": 58,
    "rek_title": "Aboriginal and Torres Strait Islander Studies Unit Publications",
    "rek_description_xsdmf_id": null,
    "rek_description": null,
    "rek_display_type_xsdmf_id": 240,
    "rek_display_type": 9,
    "rek_status_xsdmf_id": 241,
    "rek_status": 2,
    "rek_date_xsdmf_id": null,
    "rek_date": null,
    "rek_object_type_xsdmf_id": 242,
    "rek_object_type": 2,
    "rek_depositor_xsdmf_id": 7573,
    "rek_depositor": 116,
    "rek_created_date_xsdmf_id": 7571,
    "rek_created_date": "2006-12-20T21:56:09Z",
    "rek_updated_date_xsdmf_id": 7572,
    "rek_updated_date": "2019-07-05T06:00:31Z",
    "rek_file_downloads": 0,
    "rek_citation": "",
    "rek_genre_xsdmf_id": null,
    "rek_genre": null,
    "rek_genre_type_xsdmf_id": null,
    "rek_genre_type": null,
    "rek_formatted_title_xsdmf_id": null,
    "rek_formatted_title": null,
    "rek_formatted_abstract_xsdmf_id": null,
    "rek_formatted_abstract": null,
    "rek_depositor_affiliation_xsdmf_id": null,
    "rek_depositor_affiliation": null,
    "rek_thomson_citation_count": null,
    "rek_thomson_citation_count_xsdmf_id": null,
    "rek_subtype_xsdmf_id": null,
    "rek_subtype": null,
    "rek_scopus_citation_count": null,
    "rek_herdc_notes_xsdmf_id": null,
    "rek_herdc_notes": null,
    "rek_scopus_doc_type_xsdmf_id": null,
    "rek_scopus_doc_type": null,
    "rek_wok_doc_type_xsdmf_id": null,
    "rek_wok_doc_type": null,
    "rek_pubmed_doc_type_xsdmf_id": null,
    "rek_pubmed_doc_type": null,
    "rek_security_inherited": 0,
    "rek_altmetric_score": null,
    "rek_altmetric_score_xsdmf_id": null,
    "rek_altmetric_id": null,
    "rek_altmetric_id_xsdmf_id": null,
    "rek_copyright_xsdmf_id": null,
    "rek_copyright": null,
    "rek_security_policy": 5,
    "rek_datastream_policy": 5,
    "fez_record_search_key_ismemberof": [{
        "rek_ismemberof_id": 11527732,
        "rek_ismemberof_pid": "UQ:11398",
        "rek_ismemberof_xsdmf_id": 91,
        "rek_ismemberof": "UQ:12096",
        "rek_ismemberof_order": 1,
        "parent": {
            "rek_pid": "UQ:12096",
            "rek_security_policy": 5,
            "rek_datastream_policy": null
        },
        "rek_ismemberof_lookup": "Aboriginal and Torres Strait Islander Studies Unit"
    }],
    "fez_record_search_key_keywords": [{
        "rek_keywords_id": 29185812,
        "rek_keywords_pid": "UQ:11398",
        "rek_keywords_xsdmf_id": 246,
        "rek_keywords": "Torres Strait Islanders",
        "rek_keywords_order": 1
    }, {
        "rek_keywords_id": 29185813,
        "rek_keywords_pid": "UQ:11398",
        "rek_keywords_xsdmf_id": 246,
        "rek_keywords": "Aborigines",
        "rek_keywords_order": 2
    }],
    "fez_record_search_key_issn": [],
    "fez_record_search_key_doi": null,
    "fez_record_search_key_isi_loc": null,
    "fez_record_search_key_scopus_id": null,
    "fez_datastream_info": [],
    "fez_matched_journals": null,
    "rek_display_type_lookup": "Collection",
    "rek_pubmed_doc_type_lookup": null,
    "rek_object_type_lookup": "Collection",
    "rek_scopus_doc_type_lookup": null,
    "rek_status_lookup": "Published",
    "rek_wok_doc_type_lookup": null
}, {
    "rek_pid": "UQ:3585",
    "rek_title_xsdmf_id": 58,
    "rek_title": "Gooreng Gooreng Cultural Heritage Project",
    "rek_description_xsdmf_id": 59,
    "rek_description": "Aboriginal and European cultural landscapes, places and objects.",
    "rek_display_type_xsdmf_id": 240,
    "rek_display_type": 9,
    "rek_status_xsdmf_id": 241,
    "rek_status": 2,
    "rek_date_xsdmf_id": null,
    "rek_date": null,
    "rek_object_type_xsdmf_id": 242,
    "rek_object_type": 2,
    "rek_depositor_xsdmf_id": null,
    "rek_depositor": null,
    "rek_created_date_xsdmf_id": 7571,
    "rek_created_date": "2007-02-01T10:16:07Z",
    "rek_updated_date_xsdmf_id": 7572,
    "rek_updated_date": "2019-07-05T06:00:31Z",
    "rek_file_downloads": 0,
    "rek_citation": "",
    "rek_genre_xsdmf_id": null,
    "rek_genre": null,
    "rek_genre_type_xsdmf_id": null,
    "rek_genre_type": null,
    "rek_formatted_title_xsdmf_id": null,
    "rek_formatted_title": null,
    "rek_formatted_abstract_xsdmf_id": null,
    "rek_formatted_abstract": null,
    "rek_depositor_affiliation_xsdmf_id": null,
    "rek_depositor_affiliation": null,
    "rek_thomson_citation_count": null,
    "rek_thomson_citation_count_xsdmf_id": null,
    "rek_subtype_xsdmf_id": null,
    "rek_subtype": null,
    "rek_scopus_citation_count": null,
    "rek_herdc_notes_xsdmf_id": null,
    "rek_herdc_notes": null,
    "rek_scopus_doc_type_xsdmf_id": null,
    "rek_scopus_doc_type": null,
    "rek_wok_doc_type_xsdmf_id": null,
    "rek_wok_doc_type": null,
    "rek_pubmed_doc_type_xsdmf_id": null,
    "rek_pubmed_doc_type": null,
    "rek_security_inherited": 0,
    "rek_altmetric_score": null,
    "rek_altmetric_score_xsdmf_id": null,
    "rek_altmetric_id": null,
    "rek_altmetric_id_xsdmf_id": null,
    "rek_copyright_xsdmf_id": null,
    "rek_copyright": null,
    "rek_security_policy": 5,
    "rek_datastream_policy": 5,
    "fez_record_search_key_ismemberof": [{
        "rek_ismemberof_id": 11825360,
        "rek_ismemberof_pid": "UQ:3585",
        "rek_ismemberof_xsdmf_id": 91,
        "rek_ismemberof": "UQ:12096",
        "rek_ismemberof_order": 1,
        "parent": {
            "rek_pid": "UQ:12096",
            "rek_security_policy": 5,
            "rek_datastream_policy": null
        },
        "rek_ismemberof_lookup": "Aboriginal and Torres Strait Islander Studies Unit"
    }],
    "fez_record_search_key_keywords": [{
        "rek_keywords_id": 29737177,
        "rek_keywords_pid": "UQ:3585",
        "rek_keywords_xsdmf_id": 246,
        "rek_keywords": "cultural heritage",
        "rek_keywords_order": 1
    }],
    "fez_record_search_key_issn": [],
    "fez_record_search_key_doi": null,
    "fez_record_search_key_isi_loc": null,
    "fez_record_search_key_scopus_id": null,
    "fez_datastream_info": [],
    "fez_matched_journals": null,
    "rek_display_type_lookup": "Collection",
    "rek_pubmed_doc_type_lookup": null,
    "rek_object_type_lookup": "Collection",
    "rek_scopus_doc_type_lookup": null,
    "rek_status_lookup": "Published",
    "rek_wok_doc_type_lookup": null
}, {
    "rek_pid": "UQ:3586",
    "rek_title_xsdmf_id": 58,
    "rek_title": "Mill Point Archaeological Project",
    "rek_description_xsdmf_id": 59,
    "rek_description": "Aboriginal and European cultural landscapes, places and objects.",
    "rek_display_type_xsdmf_id": 240,
    "rek_display_type": 9,
    "rek_status_xsdmf_id": 241,
    "rek_status": 2,
    "rek_date_xsdmf_id": null,
    "rek_date": null,
    "rek_object_type_xsdmf_id": 242,
    "rek_object_type": 2,
    "rek_depositor_xsdmf_id": null,
    "rek_depositor": null,
    "rek_created_date_xsdmf_id": 7571,
    "rek_created_date": "2007-02-01T10:15:31Z",
    "rek_updated_date_xsdmf_id": 7572,
    "rek_updated_date": "2019-07-05T06:00:31Z",
    "rek_file_downloads": 0,
    "rek_citation": "",
    "rek_genre_xsdmf_id": null,
    "rek_genre": null,
    "rek_genre_type_xsdmf_id": null,
    "rek_genre_type": null,
    "rek_formatted_title_xsdmf_id": null,
    "rek_formatted_title": null,
    "rek_formatted_abstract_xsdmf_id": null,
    "rek_formatted_abstract": null,
    "rek_depositor_affiliation_xsdmf_id": null,
    "rek_depositor_affiliation": null,
    "rek_thomson_citation_count": null,
    "rek_thomson_citation_count_xsdmf_id": null,
    "rek_subtype_xsdmf_id": null,
    "rek_subtype": null,
    "rek_scopus_citation_count": null,
    "rek_herdc_notes_xsdmf_id": null,
    "rek_herdc_notes": null,
    "rek_scopus_doc_type_xsdmf_id": null,
    "rek_scopus_doc_type": null,
    "rek_wok_doc_type_xsdmf_id": null,
    "rek_wok_doc_type": null,
    "rek_pubmed_doc_type_xsdmf_id": null,
    "rek_pubmed_doc_type": null,
    "rek_security_inherited": 0,
    "rek_altmetric_score": null,
    "rek_altmetric_score_xsdmf_id": null,
    "rek_altmetric_id": null,
    "rek_altmetric_id_xsdmf_id": null,
    "rek_copyright_xsdmf_id": null,
    "rek_copyright": null,
    "rek_security_policy": 5,
    "rek_datastream_policy": 5,
    "fez_record_search_key_ismemberof": [{
        "rek_ismemberof_id": 11824979,
        "rek_ismemberof_pid": "UQ:3586",
        "rek_ismemberof_xsdmf_id": 91,
        "rek_ismemberof": "UQ:12096",
        "rek_ismemberof_order": 1,
        "parent": {
            "rek_pid": "UQ:12096",
            "rek_security_policy": 5,
            "rek_datastream_policy": null
        },
        "rek_ismemberof_lookup": "Aboriginal and Torres Strait Islander Studies Unit"
    }],
    "fez_record_search_key_keywords": [{
        "rek_keywords_id": 29736577,
        "rek_keywords_pid": "UQ:3586",
        "rek_keywords_xsdmf_id": 246,
        "rek_keywords": "archaeology",
        "rek_keywords_order": 1
    }, {
        "rek_keywords_id": 29736578,
        "rek_keywords_pid": "UQ:3586",
        "rek_keywords_xsdmf_id": 246,
        "rek_keywords": "cultural heritage",
        "rek_keywords_order": 2
    }],
    "fez_record_search_key_issn": [],
    "fez_record_search_key_doi": null,
    "fez_record_search_key_isi_loc": null,
    "fez_record_search_key_scopus_id": null,
    "fez_datastream_info": [],
    "fez_matched_journals": null,
    "rek_display_type_lookup": "Collection",
    "rek_pubmed_doc_type_lookup": null,
    "rek_object_type_lookup": "Collection",
    "rek_scopus_doc_type_lookup": null,
    "rek_status_lookup": "Published",
    "rek_wok_doc_type_lookup": null
}],
    "filters": {
    "facets": {
        "Scopus document type": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Display type": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": [{
                "key": 9,
                "doc_count": 3
            }]
        },
        "Keywords": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": [{
                "key": "cultural heritage",
                "doc_count": 2
            }, {
                "key": "Aborigines",
                "doc_count": 1
            }, {
                "key": "Torres Strait Islanders",
                "doc_count": 1
            }, {
                "key": "archaeology",
                "doc_count": 1
            }]
        },
        "Scopus document type (lookup)": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Subject (lookup)": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Collection (lookup)": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": [{
                "key": "Aboriginal and Torres Strait Islander Studies Unit",
                "doc_count": 3
            }]
        },
        "Year published": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Author (lookup)": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Subject": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Journal name": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Collection": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": [{
                "key": "UQ:12096",
                "doc_count": 3
            }]
        },
        "Author": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Genre": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Subtype": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": []
        },
        "Display type (lookup)": {
            "doc_count_error_upper_bound": 0,
                "sum_other_doc_count": 0,
                "buckets": [{
                "key": "Collection",
                "doc_count": 3
            }]
        }
    }
}
}
