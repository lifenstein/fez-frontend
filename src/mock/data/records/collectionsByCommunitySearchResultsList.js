export default {
  "total": 3,
  "took": 44,
  "per_page": 20,
  "current_page": 1,
  "from": 1,
  "to": 3,
  "data": [
    {
      "rek_pid": "UQ:734289",
      "rek_title_xsdmf_id": 58,
      "rek_title": "a NEW collection test by marcelo - Admin (1)",
      "rek_description_xsdmf_id": null,
      "rek_description": null,
      "rek_display_type_xsdmf_id": 240,
      "rek_display_type": 9,
      "rek_status_xsdmf_id": 241,
      "rek_status": 2,
      "rek_date_xsdmf_id": null,
      "rek_date": null,
      "rek_object_type_xsdmf_id": 242,
      "rek_object_type": 2,
      "rek_depositor_xsdmf_id": 7573,
      "rek_depositor": 1000000406,
      "rek_created_date_xsdmf_id": 7571,
      "rek_created_date": "2019-06-26T23:04:54Z",
      "rek_updated_date_xsdmf_id": 7572,
      "rek_updated_date": "2019-07-05T06:00:31Z",
      "rek_file_downloads": 0,
      "rek_citation": "",
      "rek_genre_xsdmf_id": null,
      "rek_genre": null,
      "rek_genre_type_xsdmf_id": null,
      "rek_genre_type": null,
      "rek_formatted_title_xsdmf_id": null,
      "rek_formatted_title": null,
      "rek_formatted_abstract_xsdmf_id": null,
      "rek_formatted_abstract": null,
      "rek_depositor_affiliation_xsdmf_id": null,
      "rek_depositor_affiliation": null,
      "rek_thomson_citation_count": null,
      "rek_thomson_citation_count_xsdmf_id": null,
      "rek_subtype_xsdmf_id": null,
      "rek_subtype": null,
      "rek_scopus_citation_count": null,
      "rek_herdc_notes_xsdmf_id": null,
      "rek_herdc_notes": null,
      "rek_scopus_doc_type_xsdmf_id": null,
      "rek_scopus_doc_type": null,
      "rek_wok_doc_type_xsdmf_id": null,
      "rek_wok_doc_type": null,
      "rek_pubmed_doc_type_xsdmf_id": null,
      "rek_pubmed_doc_type": null,
      "rek_security_inherited": 0,
      "rek_altmetric_score": null,
      "rek_altmetric_score_xsdmf_id": null,
      "rek_altmetric_id": null,
      "rek_altmetric_id_xsdmf_id": null,
      "rek_copyright_xsdmf_id": null,
      "rek_copyright": null,
      "rek_security_policy": 1,
      "rek_datastream_policy": 1,
      "fez_record_search_key_ismemberof": [
        {
          "rek_ismemberof_id": 12472809,
          "rek_ismemberof_pid": "UQ:734289",
          "rek_ismemberof_xsdmf_id": 91,
          "rek_ismemberof": "UQ:734288",
          "rek_ismemberof_order": 1,
          "parent": {
            "rek_pid": "UQ:734288",
            "rek_security_policy": 1,
            "rek_datastream_policy": null
          },
          "rek_ismemberof_lookup": "a NEW community test by marcelo"
        }
      ],
      "fez_record_search_key_keywords": [
        {
          "rek_keywords_id": 30879834,
          "rek_keywords_pid": "UQ:734289",
          "rek_keywords_xsdmf_id": 246,
          "rek_keywords": "testing",
          "rek_keywords_order": 1
        }
      ],
      "fez_record_search_key_issn": [],
      "fez_record_search_key_doi": null,
      "fez_record_search_key_isi_loc": null,
      "fez_record_search_key_scopus_id": null,
      "fez_datastream_info": [],
      "fez_matched_journals": null,
      "rek_display_type_lookup": "Collection",
      "rek_pubmed_doc_type_lookup": null,
      "rek_object_type_lookup": "Collection",
      "rek_scopus_doc_type_lookup": null,
      "rek_status_lookup": "Published",
      "rek_wok_doc_type_lookup": null
    },
    {
      "rek_pid": "UQ:734291",
      "rek_title_xsdmf_id": 58,
      "rek_title": "a NEW collection test by marcelo - Public (5)",
      "rek_description_xsdmf_id": null,
      "rek_description": null,
      "rek_display_type_xsdmf_id": 240,
      "rek_display_type": 9,
      "rek_status_xsdmf_id": 241,
      "rek_status": 2,
      "rek_date_xsdmf_id": null,
      "rek_date": null,
      "rek_object_type_xsdmf_id": 242,
      "rek_object_type": 2,
      "rek_depositor_xsdmf_id": 7573,
      "rek_depositor": 1000000406,
      "rek_created_date_xsdmf_id": 7571,
      "rek_created_date": "2019-06-26T23:06:21Z",
      "rek_updated_date_xsdmf_id": 7572,
      "rek_updated_date": "2019-07-17T23:00:51Z",
      "rek_file_downloads": 0,
      "rek_citation": "",
      "rek_genre_xsdmf_id": null,
      "rek_genre": null,
      "rek_genre_type_xsdmf_id": null,
      "rek_genre_type": null,
      "rek_formatted_title_xsdmf_id": null,
      "rek_formatted_title": null,
      "rek_formatted_abstract_xsdmf_id": null,
      "rek_formatted_abstract": null,
      "rek_depositor_affiliation_xsdmf_id": null,
      "rek_depositor_affiliation": null,
      "rek_thomson_citation_count": null,
      "rek_thomson_citation_count_xsdmf_id": null,
      "rek_subtype_xsdmf_id": null,
      "rek_subtype": null,
      "rek_scopus_citation_count": null,
      "rek_herdc_notes_xsdmf_id": null,
      "rek_herdc_notes": null,
      "rek_scopus_doc_type_xsdmf_id": null,
      "rek_scopus_doc_type": null,
      "rek_wok_doc_type_xsdmf_id": null,
      "rek_wok_doc_type": null,
      "rek_pubmed_doc_type_xsdmf_id": null,
      "rek_pubmed_doc_type": null,
      "rek_security_inherited": 0,
      "rek_altmetric_score": null,
      "rek_altmetric_score_xsdmf_id": null,
      "rek_altmetric_id": null,
      "rek_altmetric_id_xsdmf_id": null,
      "rek_copyright_xsdmf_id": null,
      "rek_copyright": null,
      "rek_security_policy": 5,
      "rek_datastream_policy": 5,
      "fez_record_search_key_ismemberof": [
        {
          "rek_ismemberof_id": 12472811,
          "rek_ismemberof_pid": "UQ:734291",
          "rek_ismemberof_xsdmf_id": 91,
          "rek_ismemberof": "UQ:734288",
          "rek_ismemberof_order": 1,
          "parent": {
            "rek_pid": "UQ:734288",
            "rek_security_policy": 1,
            "rek_datastream_policy": null
          },
          "rek_ismemberof_lookup": "a NEW community test by marcelo"
        }
      ],
      "fez_record_search_key_keywords": [
        {
          "rek_keywords_id": 30879836,
          "rek_keywords_pid": "UQ:734291",
          "rek_keywords_xsdmf_id": 246,
          "rek_keywords": "testing",
          "rek_keywords_order": 1
        }
      ],
      "fez_record_search_key_issn": [],
      "fez_record_search_key_doi": null,
      "fez_record_search_key_isi_loc": null,
      "fez_record_search_key_scopus_id": null,
      "fez_datastream_info": [],
      "fez_matched_journals": null,
      "rek_display_type_lookup": "Collection",
      "rek_pubmed_doc_type_lookup": null,
      "rek_object_type_lookup": "Collection",
      "rek_scopus_doc_type_lookup": null,
      "rek_status_lookup": "Published",
      "rek_wok_doc_type_lookup": null
    },
    {
      "rek_pid": "UQ:734290",
      "rek_title_xsdmf_id": 58,
      "rek_title": "a NEW collection test by marcelo - Staff and Students (4)",
      "rek_description_xsdmf_id": null,
      "rek_description": null,
      "rek_display_type_xsdmf_id": 240,
      "rek_display_type": 9,
      "rek_status_xsdmf_id": 241,
      "rek_status": 2,
      "rek_date_xsdmf_id": null,
      "rek_date": null,
      "rek_object_type_xsdmf_id": 242,
      "rek_object_type": 2,
      "rek_depositor_xsdmf_id": 7573,
      "rek_depositor": 1000000406,
      "rek_created_date_xsdmf_id": 7571,
      "rek_created_date": "2019-06-26T23:05:41Z",
      "rek_updated_date_xsdmf_id": 7572,
      "rek_updated_date": "2019-07-05T06:00:31Z",
      "rek_file_downloads": 0,
      "rek_citation": "",
      "rek_genre_xsdmf_id": null,
      "rek_genre": null,
      "rek_genre_type_xsdmf_id": null,
      "rek_genre_type": null,
      "rek_formatted_title_xsdmf_id": null,
      "rek_formatted_title": null,
      "rek_formatted_abstract_xsdmf_id": null,
      "rek_formatted_abstract": null,
      "rek_depositor_affiliation_xsdmf_id": null,
      "rek_depositor_affiliation": null,
      "rek_thomson_citation_count": null,
      "rek_thomson_citation_count_xsdmf_id": null,
      "rek_subtype_xsdmf_id": null,
      "rek_subtype": null,
      "rek_scopus_citation_count": null,
      "rek_herdc_notes_xsdmf_id": null,
      "rek_herdc_notes": null,
      "rek_scopus_doc_type_xsdmf_id": null,
      "rek_scopus_doc_type": null,
      "rek_wok_doc_type_xsdmf_id": null,
      "rek_wok_doc_type": null,
      "rek_pubmed_doc_type_xsdmf_id": null,
      "rek_pubmed_doc_type": null,
      "rek_security_inherited": 0,
      "rek_altmetric_score": null,
      "rek_altmetric_score_xsdmf_id": null,
      "rek_altmetric_id": null,
      "rek_altmetric_id_xsdmf_id": null,
      "rek_copyright_xsdmf_id": null,
      "rek_copyright": null,
      "rek_security_policy": 4,
      "rek_datastream_policy": 5,
      "fez_record_search_key_ismemberof": [
        {
          "rek_ismemberof_id": 12472810,
          "rek_ismemberof_pid": "UQ:734290",
          "rek_ismemberof_xsdmf_id": 91,
          "rek_ismemberof": "UQ:734288",
          "rek_ismemberof_order": 1,
          "parent": {
            "rek_pid": "UQ:734288",
            "rek_security_policy": 1,
            "rek_datastream_policy": null
          },
          "rek_ismemberof_lookup": "a NEW community test by marcelo"
        }
      ],
      "fez_record_search_key_keywords": [
        {
          "rek_keywords_id": 30879835,
          "rek_keywords_pid": "UQ:734290",
          "rek_keywords_xsdmf_id": 246,
          "rek_keywords": "testing",
          "rek_keywords_order": 1
        }
      ],
      "fez_record_search_key_issn": [],
      "fez_record_search_key_doi": null,
      "fez_record_search_key_isi_loc": null,
      "fez_record_search_key_scopus_id": null,
      "fez_datastream_info": [],
      "fez_matched_journals": null,
      "rek_display_type_lookup": "Collection",
      "rek_pubmed_doc_type_lookup": null,
      "rek_object_type_lookup": "Collection",
      "rek_scopus_doc_type_lookup": null,
      "rek_status_lookup": "Published",
      "rek_wok_doc_type_lookup": null
    }
  ],
  "filters": {
    "facets": {
      "Scopus document type": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Display type": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": [
          {
            "key": 9,
            "doc_count": 3
          }
        ]
      },
      "Keywords": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": [
          {
            "key": "testing",
            "doc_count": 3
          }
        ]
      },
      "Scopus document type (lookup)": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Subject (lookup)": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Collection (lookup)": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": [
          {
            "key": "a NEW community test by marcelo",
            "doc_count": 3
          }
        ]
      },
      "Year published": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Author (lookup)": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Subject": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Journal name": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Collection": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": [
          {
            "key": "UQ:734288",
            "doc_count": 3
          }
        ]
      },
      "Author": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Genre": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Subtype": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": []
      },
      "Display type (lookup)": {
        "doc_count_error_upper_bound": 0,
        "sum_other_doc_count": 0,
        "buckets": [
          {
            "key": "Collection",
            "doc_count": 3
          }
        ]
      }
    }
  }
}