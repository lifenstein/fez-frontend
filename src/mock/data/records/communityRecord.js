import { hydrateMock } from '../../../helpers/general';

const communityRecord = {
    "rek_pid": "UQ:3883",
    "rek_title": "The University of Queensland Library",
    "rek_display_type": 11,
    "rek_status": 2,
    "rek_object_type": 1,
    "rek_depositor": 112,
    "rek_created_date": "2006-12-05T15:29:26Z",
    "rek_updated_date": "2020-12-18T04:12:20Z",
    "rek_file_downloads": 0,
    "rek_citation": "<a class=\"citation_community\" title=\"Click to view Fedora Community Display The University of Queensland Library\" href=\"/view/UQ%3A3883\">The University of Queensland Library</a>",
    "rek_security_inherited": 0,
    "rek_security_policy": 5,
    "fez_record_search_key_keywords": ["library"],
    "rek_display_type_lookup": "Community",
    "rek_object_type_lookup": "Community",
    "rek_status_lookup": "Published",
    "rek_wok_doc_type_lookup": "Abstract of Published Item",
};
export default hydrateMock(communityRecord);
