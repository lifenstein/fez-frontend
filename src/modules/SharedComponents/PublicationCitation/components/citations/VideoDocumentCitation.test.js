import React from 'react';
import VideoDocumentCitation from './VideoDocumentCitation';
import { videoDocument } from 'mock/data/testing/records';
import { rtlRender } from 'test-utils';

function setup(testProps = {}) {
    const props = {
        classes: {},
        ...testProps,
        publication: testProps.publication || {},
    };
    return rtlRender(<VideoDocumentCitation {...props} />);
}

describe('VideoDocumentCitation renders ', () => {
    it('component with empty publication', () => {
        const { container } = setup();
        expect(container).toMatchSnapshot();
    });

    it('component with a mock espace record', () => {
        const { container } = setup({ publication: videoDocument });
        expect(container).toMatchSnapshot();
    });
});
