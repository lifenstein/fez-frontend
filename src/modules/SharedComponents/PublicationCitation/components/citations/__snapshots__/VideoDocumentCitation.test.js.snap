// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoDocumentCitation renders  component with a mock espace record 1`] = `
<div>
  <div
    class="citationContent citationVideo"
  >
    <span
      class="citationAuthors"
    >
      <span
        class="citationAuthor"
        data-testid="rek-author-0"
      >
        Walters, S.
      </span>
       
    </span>
    <span
      class="citationDate"
    >
      (
      2002
      ).
    </span>
    <span
      class="citationTitle"
    >
       
      Video 12: 1) Discovery History and Geology of the Cannington Ag-Pb-Zn Deposit 2) Advances in Micro-Analytical Technologies â€” What a grain can tell you
      .
    </span>
    <span
      class="citationSeries"
    >
       
      SEG Special Publication Video Series
      .
    </span>
  </div>
</div>
`;

exports[`VideoDocumentCitation renders  component with empty publication 1`] = `
<div>
  <div
    class="citationContent citationVideo"
  >
    <span
      class="citationAuthors empty"
    />
    <span
      class="citationDate empty"
    />
    <span
      class="citationTitle empty"
    />
    <span
      class="citationSeries empty"
    />
  </div>
</div>
`;
